test/support/builders/stats.js

```diff
@@ -100,6 +106,11 @@ class WinStatsBuilder {
     return this
   }
 
+  mtime (newMtime /*: Date */) /*: this */ {
+    this.winStats.mtime = new Date(newMtime)
+    return this
```

Comment:
Having 2 classes implementing `StatsBuilder` actually favors duplication. My bad.
Out of the scope of this PR anyway.